spec/features/generic_file_new_form_spec.rb

```diff
@@ -1,88 +1,74 @@
 require 'spec_helper'
```

Comment:
This is a really bad test. First its a form test, this should be testing against form class. But there isn't a generic_file_new_form class, so we are doing a integration test instead? Let's move this in the right folder then....